blockchain/flowkit_test.go

```diff
@@ -68,3 +69,50 @@ func Test_NewFlowkit(t *testing.T) {
 		//assert.Equal(t, accountStorage.Account.Address.Bytes(), accountList[i].Address.Bytes())
 	}
 }
+
+func Test_FlowJsonExport(t *testing.T) {
```

Comment:
What about accounts with the emulator account? Should that be added to the json?

Reply:
They are added already if you mean the accounts created on emulator such as:

0x01, 0x02, 0x03, 0x04, 0x05, emulator-accounts, and Service Account